README.md

```diff
@@ -132,18 +132,54 @@ All classes are considered internal and are not subject to semver.
 
 ## Installation
 
-The recommended installation method is through [Composer](https://getcomposer.org/).
-Simply run
-
-    composer require felixfbecker/language-server
-
-and you will get the latest stable release and all dependencies.  
-Running `composer update` will update the server to the latest non-breaking version.
-
-After installing the language server and its dependencies,
-you must parse the stubs for standard PHP symbols and save the index for fast initialization.
-
-     composer run-script --working-dir=vendor/felixfbecker/language-server parse-stubs
+The recommended installation method is through [Composer](https://getcomposer.org/).  Follow the following installation instructions for local or global installation after installing composer on your system.
+
+### Local Installation
+Create a directory for php-language-server. Create a composer.json file in it, with the following contents:
+
+```
+{
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "felixfbecker/language-server": "v5.4.0"
+    },
+    "scripts": {
+        "parse-stubs": "LanguageServer\\ComposerScripts::parseStubs",
+        "post-update-cmd": "@parse-stubs",
+        "post-install-cmd": "@parse-stubs"
+    }
+}
+```
+Then, in the directory, run the following commands:
+
+```
+composer require felixfbecker/language-server
+```
+
+### Global installation
+Before installing php-language-server, make sure your ~/.config/composer/composer.json includes the lines below. The settings apply to all globally installed Composer packages, so proceed with caution. If you do not want to edit your global Composer configuration, see the section for local installation above.
```

Comment:
At least on my system (Mac with homebrew installed php and composer), the global config is in `~/.composer/composer.json` rather than `~/.config/composer/composer.json`.

Reply:
More generally, use `$COMPOSER_HOME` if it's set.  Refer to https://getcomposer.org/doc/03-cli.md#composer-home

If it's not set, note that a hard-coded fallback could be error-prone, as the default value is quite variable depending on the OS (and version thereof) you're running.